activerecord/test/schema/schema.rb

```diff
@@ -310,6 +310,11 @@ def except(adapter_names_to_exclude)
   create_table :eyes, force: true do |t|
   end
 
+  create_table :flowers, force: true do |t|
```

Comment:
Can we check this without adding a new model? Perhaps just temporarily add the timestamps to an existing table?